ci/docker/Dockerfile

```diff
@@ -57,6 +57,7 @@ RUN chmod +x /usr/local/bin/tini \
     python3-setuptools \
     unzip \
     vim \
+    make \
```

Comment:
Only because the rest already is, can we alphabetize the list?

Reply:
Done